test/integration/test-amp-ad-nativo.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
```

Comment:
@erwinmombay   In 2017, are we required to update this?  Why not just use multiple years instead? 2015-2016.